android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlBroadcastReceiver.java

```diff
@@ -49,6 +49,7 @@
 
 import androidx.annotation.CallSuper;
 
+import com.smartdevicelink.proxy.rpc.VehicleType;
```

Comment:
@kostyaBoss Do you need to make changes also in `SdlReceiver.java`?

Per the proposal, there needs to be a method `private boolean IsVehicleTypeSupported(Intent intent)...`

Reply:
@santhanamk it was an example on how the developer can handle it. I didn't include this into the original or the decisions on handling the parameters are up to dev